src/platform/site-wide/sass/style.scss

```diff
@@ -28,5 +28,6 @@
 @import "modules/m-veteran-banner";
 @import "modules/m-cta-widget";
 @import "modules/m-side-nav";
+@import "modules/m-responsive-table";
```

Comment:
It's not clear why this is added to the site-wide css instead of to the GIBCT css.

Reply:
This is because from https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203#issuecomment-679138608 it seemed that the css for a responsive tables was more useful as available for platform wide use instead of a ResponsiveTable component.

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/203 mentioned the work for https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.jsx as being an example for creating ResponsiveTables so pulled the css used for it

This PR is part of the ongoing work related to https://github.com/department-of-veterans-affairs/va.gov-team/issues/11055 for the GIBCT


The css file might need cleaning up to remove attributes that are not needed for the required responsive behavior.

Reply:
Cleaned up `modules/m-responsive-table` to only include required CSS 

This causes 

![Screen Shot 2020-09-18 at 9 36 34 AM](https://user-images.githubusercontent.com/1094999/93603759-71eb2a00-f992-11ea-9d23-e2d08f76c9c9.png)

to become below on screen widths under 600px

![image](https://user-images.githubusercontent.com/1094999/93603737-6c8ddf80-f992-11ea-896f-443f57896856.png)